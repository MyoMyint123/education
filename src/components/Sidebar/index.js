import React from "react";
import { SidebarMenu } from "../Menu";

const menuItems = [
  { name: "Dashboard", link: "/dashboard", iconType: "tachometer" },
  { name: "School", link: "/schools", iconType: "graduation-cap" },
  { name: "User", link: "/users", iconType: "users" },
  { name: "Setting", link: "/setting", iconType: "cog" }
];

const Sidebar = () => (
  <nav className="sidebar bg-dark">
    <SidebarMenu menus={menuItems} />
  </nav>
);

export default Sidebar;
