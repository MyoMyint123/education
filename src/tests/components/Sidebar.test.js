import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from '../../components/Sidebar';

test('should render Sidebar correctly', () => {
  const wrapper = shallow(<Sidebar />);
  expect(wrapper).toMatchSnapshot();
});
