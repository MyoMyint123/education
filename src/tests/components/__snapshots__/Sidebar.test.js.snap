// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Sidebar correctly 1`] = `
<nav
  className="sidebar bg-dark"
>
  <SidebarMenu
    menus={
      Array [
        Object {
          "iconType": "tachometer",
          "link": "/dashboard",
          "name": "Dashboard",
        },
        Object {
          "iconType": "graduation-cap",
          "link": "/schools",
          "name": "School",
        },
        Object {
          "iconType": "users",
          "link": "/users",
          "name": "User",
        },
        Object {
          "iconType": "cog",
          "link": "/setting",
          "name": "Setting",
        },
      ]
    }
  />
</nav>
`;
